### DISCUSSION TITLE
xDrip displaying only every second CGM reading

### DISCUSSION THREAD

**Discussion by kl4mpt** [NightscoutFoundation/xDrip#3397] [Q&A]
So, I use POCTech CGM. I transfer data from their app with AndroidAPS (PumpControl). It receives and shows in logs every reading (once every 3 minutes) and it is set up to send readings to xDrip. It shows that every 3 minutes, as it should, it sends data to xDrip, but in the app itself I see only every second reading (so, every 6 minutes I get a new one on the graph). I use source as a follower and also set xDrip to be master for sending my readings, could the problem be there, maybe interference? But I don't think so, because AAPS broadcasts locally and, as I said, every second reading displays flawlessly. So, what can I do to bring up those readings?

**Comment by Navid200** [comment]
> It receives and shows in logs every reading (once every 3 minutes)  

Every 3 minutes, not 5 minutes?
Is there a user manual or any statement from the company you can link that states that?

**Reply by Navid200** [reply]
OK, I can see it: https://www.poctechcorp.com/en/contents/293/5836.html
![Screenshot 2024-03-20 193848](https://github.com/NightscoutFoundation/xDrip/assets/51497406/dbc6451c-bacd-4617-9bc9-c9140e4a1d10)

**Reply by kl4mpt** [reply]
The sensor just was created like this (https://cgm-poctech.com/wp-content/uploads/2023/10/POCTech_User_Manual_uputa_A4_HR.pdf ; you can see on the page 89 of the manual)
And also, want to say that i use CT14 sensor (like in the sent manual), not the CT-100B that is listed globally when you search for POCTech CGMs

**Reply by Navid200** [reply]
xDrip operates at a one reading per 5 minutes rate.
There is a test feature that enables one reading per minute for only one source in engineering mode: https://github.com/NightscoutFoundation/xDrip/issues/2680

This falls in the same category that will require new features.
If you read the very first post in that thread from @jamorham, you will see this:
> For example there will likely be a smoothing emitter which can receive data at any interval and then emit to services that can only accept 5 minute readings at that rate. This is work in progress still.  
  
So, this is not something that is currently possible.  Let's see if jamorham has any advice.
